ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -994,7 +1001,9 @@ private[platform] object JdbcLedgerDao {
       * Performance optimization for transactions that don't
       * require strong durability guarantees.
       */
-    protected[JdbcLedgerDao] def enableAsyncCommit(implicit conn: Connection): Unit
+    protected[JdbcLedgerDao] def enableAsyncCommit(logger: ContextualizedLogger)(
```

Comment:
Rather than passing a `logger`, why not make this part of the class?

Reply:
@SamirTalwar-DA , I'll actually remove the logging statement entirely. It's cluttering all the tests output without great benefit. I'll just leave and assert against the one at class initialisation